code42_connector.py

```diff
@@ -136,14 +173,16 @@ def main():
         connector.print_progress_message = True
 
         if session_id is not None:
-            in_json['user_session_token'] = session_id
-            connector._set_csrf_info(csrftoken, headers['Referer'])
+            in_json["user_session_token"] = session_id
+            if csrftoken and headers:
+                connector._set_csrf_info(csrftoken, headers["Referer"])
 
-        ret_val = connector._handle_action(json.dumps(in_json), None)
+        json_string = json.dumps(in_json)
```

Comment:
Every change in this part is just making strings use double quotes besides this one,
I separated out this variable to attempt at making the debugging experience better